### PR TITLE
Show client and spouse last four of SSN/ITIN on outbound call page

### PR DESCRIPTION
**Pull request by jenny-heath**
[story](https://www.pivotaltracker.com/story/show/177729431) had minimal instruction so this is what it looks like:

image

apologies for the quality of my spanish translations :(

### CODE REVIEW

**Review by coltborg — APPROVED** [review]
LGTM! :shipit:

**Comment by alaroia-cfa** [comment]
@jenny-heath If we're showing the client SSN/ITIN last 4, I'd like to see us record an AccessLog record; see the `AccessLog` model for details. Not strictly a blocker I suppose, but I would really like to see it.

**Review by coltborg — APPROVED** [review]
Reviewed the access log code. Looks good to me, thanks for adding the test as well!

File: app/controllers/hub/outbound_calls_controller.rb [hunk 1]
```diff
@@ -17,6 +17,14 @@ def create
 
     def show
       @outbound_call = OutboundCall.find(params[:id])
+      AccessLog.create(
```
Comment (alaroia-cfa) [inline app/controllers/hub/outbound_calls_controller.rb:20]: Consider `.create!`. Not a blocker.

**Review by alaroia-cfa — APPROVED** [review]
Agree with @coltborg !

**Pull request merged**